pages/reset/[token].jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { withRouter } from 'next/router'
+
+import styles from './styles.module.css'
+
+import { ChangePassword } from 'components/forms'
+
+const ResetStep2 = ({ router }) => (
+  <ChangePassword
+    tag="div"
+    className={styles.resetContainer}
+    email={router.query.email}
+    token={router.query.token}
+  />
+)
+
+export default withRouter(ResetStep2)
```

Comment:
I am struggling to find a better name but the following is the best I can imagine at the moment. When numbers appear in the name it's a sign that something is not good.

```suggestion
const CreatePassword = ({ router }) => (
  <ChangePassword
    tag="div"
    className={styles.resetContainer}
    email={router.query.email}
    token={router.query.token}
  />
)

export default withRouter(CreatePassword)
```

Why do not you use `useRouter` hook instead? Just curious 🧐 Your option maybe more flexible

Reply:
No reason I just forgot that it existed.